__tests__/env-tests/staging.test.js

```diff
@@ -1,10 +1,10 @@
-process.env.NODE_ENV = 'beta';
+process.env.NODE_ENV = 'staging';
```

Comment:
I like `staging` better than `beta` for sure. It's definitely a better description of what it actually is.